src/language-css/parser-postcss.js

```diff
@@ -32,6 +32,12 @@ function parseValueNodes(nodes) {
   for (let i = 0; i < nodes.length; ++i) {
     const node = nodes[i];
 
+    if (node.type === "func" && node.value === "selector") {
+      const selector = parseSelector(stringifyNode(node.group.groups[0]));
```

Comment:
This groups[0] not right `selector()` `selector(.a,.b)`

Reply:
Argh, yep, fixed.